### ISSUE TITLE
Build failure with Vector Compute disabled

### ISSUE THREAD

**Issue by eero-t** [intel/intel-graphics-compiler#143]
When building IGC with -DIGC_BUILD__VC_ENABLED=OFF, build fails to:
```
../IGC/AdaptorOCL/dllInterfaceCompute.cpp:1533:12: error: could not convert 'false' from 'bool' to 'std::error_code'
 1533 |     return false;
      |            ^~~~~
      |            |
      |            bool
```

Note: Changing "return false" to "return {}" inside IGC_VC_DISABLED ifdef fixes the build failure.

**Comment by kvladimi** [comment]
Fixed in 2a1f21ef1899fd7b6810a37a90a274b709be6686

**Comment by eero-t** [comment]
Latest "igc-1.0.4944" release from 2 days ago builds fine with VC disabled (previous release didn't) -> VERIFIED

This will help in debugging compute-runtime issue when VC is disabled in IGC: https://github.com/intel/compute-runtime/issues/346
